content/en/tracing/trace_collection/custom_instrumentation/dotnet/otel.md

```diff
@@ -17,33 +17,108 @@ further_reading:
 ---
 
-{{% otel-custom-instrumentation %}}
+{{% otel-custom-instrumentation-lang %}}
 
-## Requirements and limitations
+## Setup
 
-- Datadog .NET tracing library `dd-trace-dotnet` version 2.21.0 or greater.
+To configure OpenTelemetry to use the Datadog trace provider:
 
-The following OpenTelemetry features implemented in the Datadog library as noted:
+1. Add your desired manual OpenTelemetry instrumentation to your .NET code following the [OpenTelemetry .NET Manual Instrumentation documentation][5]. **Important!** Where those instructions indicate that your code should call the OpenTelemetry SDK, call the Datadog tracing library instead.
 
-| Feature                               | Support notes                                           |
-|---------------------------------------|---------------------------------------------------------|
-| OpenTelemetry Context propagation     | [W3C Trace Context and Datadog header formats][9] are enabled by default.  |
-| [Span processors][2]                  | Unsupported                                             |
-| [Span Exporters][3]                   | Unsupported                                             |
-| Trace/span [ID generators][4]         | ID generation is performed by the tracing library, with support for [128-bit trace IDs][12].  |
+2. Install the Datadog .NET tracing library and enable the tracer for your [.NET Framework service][10] or your [.NET Core (and .NET 5+) service][11]. **Beta:** You can optionally do this with [Single Step APM Instrumentation][13].
 
+3. Set `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 
-## Configuring OpenTelemetry to use the Datadog trace provider
+4. Run your application.
 
-1. Add your desired manual OpenTelemetry instrumentation to your .NET code following the [OpenTelemetry .NET Manual Instrumentation documentation][5]. **Important!** Where those instructions indicate that your code should call the OpenTelemetry SDK, call the Datadog tracing library instead.
+Datadog combines these OpenTelemetry spans with other Datadog APM spans into a single trace of your application. It also supports [OpenTelemetry instrumentation libraries][8].
 
-2. Install the Datadog .NET tracing library and enable the tracer for your [.NET Framework service][10] or your [.NET Core (and .NET 5+) service][11]. **Beta:** You can optionally do this with [One-Step APM Instrumentation][13].
+## Creating custom spans
 
-3. Set `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
+To create custom spans within an existing trace context:
 
-4. Run your application.
+```csharp
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+// Start a new span
+using (Activity? activity = Telemetry.ActivitySource.StartActivity("<RESOURCE NAME>"))
+            {
+  activity?.SetTag("operation.name", "custom-operation");
+               // Do something
+            }
+
+```
+
+## Creating new spans
+
+To manually create new spans that start a new, independent trace:
```

Comment:
For .NET this line below should be swapped for the other to match the code snippets:
> To manually create new spans that start a new, independent trace:
---
> To create custom spans within an existing trace context:

Reply:
@link04 - Thanks for taking a look and good catch. I swapped these sentences to match the code snippets better. 🙂